src/main/kotlin/gg/essential/universal/UDesktop.kt

```diff
@@ -70,12 +71,7 @@ object UDesktop {
 
     private fun openSystemSpecific(file: String): Boolean {
```

Comment:
Github won't let me comment on the on `browse` but perhaps we should a comment warning that the call can block for several seconds as a result of `process.waitFor(3, TimeUnit.SECONDS)` in `runCommand`

Reply:
Idk if that needs to be explicitly stated. It should go without saying that opening a file in an external program and returning whether the operation was successful can take some time, just like any other IO.